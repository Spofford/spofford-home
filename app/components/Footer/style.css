footer.footer-container {
  height:142px;
  border-top:1px solid black;
}

footer h2, footer h4 {
  text-align:center;
}

footer h4 {
  margin-top:5px;
}

footer h2 {
  line-height: 1.5em;
  font-size:1.2em;
}
