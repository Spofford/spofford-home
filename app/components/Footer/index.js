import React from "react";
import FontAwesome from "react-fontawesome";
import cssModules from 'react-css-modules';
import style from "./style.css";

export const Footer = () => {
  return (
    <footer className='footer-container'>
      <h2>Spofford Design, Inc.</h2>
      <div>
        <h4>Copyright 2018</h4>
      </div>
    </footer>
  )
};

export default cssModules(Footer, style);
