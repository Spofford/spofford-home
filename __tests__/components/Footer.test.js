import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

import Footer from '../../app/components/Footer';

describe('<Footer />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<Footer />);
      // const component = wrapper.dive();

      expect(toJson(wrapper)).toMatchSnapshot();
    });

    // test to render model from contentful
  });
});
