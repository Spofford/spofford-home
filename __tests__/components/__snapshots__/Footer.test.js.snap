// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render() renders the component 1`] = `
<footer
  className="footer-container"
>
  <h2>
    Spofford Design, Inc.
  </h2>
  <div>
    <h4>
      Copyright 2018
    </h4>
  </div>
</footer>
`;
